pkg/parsers/filters/parse.go

```diff
@@ -7,16 +7,17 @@ import (
 	"strings"
 )
 
+// Args arguments as map key:{array}
```

Comment:
This should probably give more about the intent of this type declaration. Why not just use `map[string][]string` directly? What kind of "args" are these?